docs/src/APIs/Common/TurbulenceClosures.md

```diff
@@ -12,6 +12,8 @@ TurbulenceClosures
 
 ```@docs
 TurbulenceClosureModel
+hyperdiff_enthalpy_and_visc_flux
```

Comment:
I'd call this `hyperdiff_enthalpy_and_momentum_flux` throughout (clearer than viscous flux, and parallel to enthalpy).